### PR TITLE
feat(config): add/update MCOHome config files for v5 devices

### PR DESCRIPTION
**Pull request by jcam**
This adds support for the following v5 mco home devices:
MH-C421 - Shutters (cover device)
MH-DT411 - gen5 touch dimmer
MH-S314 - gen5 4 button touch switch
MH-S411 - gen5 1 button touch switch
MH-S412 - gen5 2 button touch switch

### CODE REVIEW

File: packages/config/config/devices/0x015f/mh-c421.json [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+			"minValue": 0,
+			"maxValue": 10,
+			"defaultValue": 10,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"9": {
+			"label": "Demo Trip and Auto Calibration",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-c421.json]: Any idea what this is that we could describe in a description?

File: packages/config/config/devices/0x015f/mh-s314-1502.json [hunk 2]
```diff
@@ -0,0 +1,703 @@
...
+			"minValue": 0,
+			"maxValue": 65535,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"8": {
+			"label": "Basic CC Integration Setting",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-s314-1502.json]: Any idea what this is, or what the modes do? Can we make these labels more descriptive? Add a description to the parameter if not? I am only marking things once so please carry through.

File: packages/config/config/devices/0x015f/mh-s411-1502.json [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+				}
+			]
+		},
+		"6": {
+			"label": "Key On Duration",
+			"description": "0 for infinite",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 65535,
+			"defaultValue": 0,
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-s411-1502.json:131]: Is there a unit we can put here? I think there are some others like this.

File: packages/config/config/devices/0x015f/mh-s411-1502.json [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"8": {
+			"label": "Basic CC Integration Setting",
+			"description": "See Manual",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-s411-1502.json]: As above. Can we make the label descriptive? Add a description? (If not, delete the See manual description)

**Review by blhoward2 — CHANGES_REQUESTED** [review]
Looking great!

File: packages/config/config/devices/0x015f/mh-c421.json [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+					"label": "Activate",
+					"value": 1
+				}
+			]
+		},
+		"11": {
+			"label": "Min Level Shutter Close",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 98,
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-c421.json:144]: Is this really 98 and not 99?

File: packages/config/config/devices/0x015f/mh-s314.json [hunk 6]
```diff
@@ -103,12 +134,13 @@
 			"allowManualEntry": true
 		},
 		"255": {
-			"label": "Factory setting",
+			"label": "Factory Setting",
 			"description": "85 restore factory setting",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-s314.json]: This doesn't make sense. 85 isn't allowed within the range. Could you verify? Usually we add an option that says Normal Operation and value 0 (that does nothing), then add an option with the reset value.

**Review by blhoward2 — CHANGES_REQUESTED** [review]
Forgot a couple. Note that there are lint errors for two of the parameters that need to be unsigned.

File: packages/config/config/devices/0x015f/mh-c421.json (hunk 5 above)
Comment (jcam) [reply packages/config/config/devices/0x015f/mh-c421.json:144]: yep

**Review by blhoward2 — DISMISSED** [review]
Looks good to me. Please request a review from @AlCalzone so he can give final approval and merge.

**Comment by jcam** [comment]
Oddly, I can't request a review. But I imagine he'll see your comment tagging him :)

@AlCalzone: Happy to rework whatever on these. Your conditional parameters might end up being something we want here.

**Comment by blhoward2** [comment]
You can’t request a review at the top under Reviewers? I did it

File: packages/config/config/devices/0x015f/mh-c421.json [hunk 7]
```diff
@@ -0,0 +1,238 @@
...
+			"minValue": 1,
+			"maxValue": 32767,
+			"defaultValue": 720,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"2": {
+			"label": "KWH Meter Report Period",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x015f/mh-c421.json]:
```suggestion
			"label": "kWh Meter Report Period",
```

File: packages/config/config/devices/0x015f/mh-c421.json [hunk 8]
```diff
@@ -0,0 +1,238 @@
...
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 1
+				}
+			]
+		},
+		"8": {
+			"label": "LED BackLight Brightness",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x015f/mh-c421.json]:
```suggestion
			"label": "LED Backlight Brightness",
```

File: packages/config/config/devices/0x015f/mh-dt311_411.json [hunk 9]
```diff
@@ -0,0 +1,531 @@
...
+					"value": 0
+				},
+				{
+					"label": "Enabled",
+					"value": 1
+				}
+			]
+		},
+		"15[0x02]": {
+			"label": "Energy Meter Report KWH",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x015f/mh-dt311_411.json]:
```suggestion
			"label": "Energy Meter Report kWh",
```

File: packages/config/config/devices/0x015f/mh-dt311_411.json [hunk 10]
```diff
@@ -0,0 +1,531 @@
...
+				},
+				{
+					"label": "Enabled",
+					"value": 1
+				}
+			]
+		},
+		"32": {
+			"label": "Ampere Alarm Threshold",
+			"unit": "0.01A",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x015f/mh-dt311_411.json]:
```suggestion
			"unit": "0.01 A",
```

File: packages/config/config/devices/0x015f/mh-dt311_411.json [hunk 11]
```diff
@@ -0,0 +1,531 @@
...
+			"maxValue": 5000,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"33": {
+			"label": "Voltage Alarm Threshold",
+			"unit": "0.1V",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x015f/mh-dt311_411.json]:
```suggestion
			"unit": "0.1 V",
```

File: packages/config/config/devices/0x015f/mh-dt311_411.json [hunk 12]
```diff
@@ -0,0 +1,531 @@
...
+			"maxValue": 10000,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"34": {
+			"label": "Wattage Alarm Threshold",
+			"unit": "0.1W",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x015f/mh-dt311_411.json]:
```suggestion
			"unit": "0.1 W",
```

File: packages/config/config/devices/0x015f/mh-dt311_411.json [hunk 13]
```diff
@@ -0,0 +1,531 @@
...
+				},
+				{
+					"label": "Enabled",
+					"value": 1
+				}
+			]
+		},
+		"64": {
+			"label": "Overloaded Amps Detected",
+			"unit": "0.01A",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x015f/mh-dt311_411.json]:
```suggestion
			"unit": "0.01 A",
```

File: packages/config/config/devices/0x015f/mh-dt311_411.json [hunk 14]
```diff
@@ -0,0 +1,531 @@
...
+			"maxValue": 5000,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": true,
+			"writeOnly": false,
+			"allowManualEntry": false
+		},
+		"65": {
+			"label": "Overloaded Volts Detected",
+			"unit": "0.1V",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x015f/mh-dt311_411.json]:
```suggestion
			"unit": "0.1 V",
```

File: packages/config/config/devices/0x015f/mh-dt311_411.json [hunk 15]
```diff
@@ -0,0 +1,531 @@
...
+			"maxValue": 10000,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": true,
+			"writeOnly": false,
+			"allowManualEntry": false
+		},
+		"66": {
+			"label": "Overloaded Watts Detected",
+			"unit": "0.1W",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x015f/mh-dt311_411.json]:
```suggestion
			"unit": "0.1 W",
```
Please check the other units too :)

**Review by AlCalzone — CHANGES_REQUESTED** [review]

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**